test-one/src/index-test.js

```diff
@@ -1,13 +1,55 @@
 import { go } from "./index";
 import * as logger from "./helpers/logger";
 
-test("logged welcome message", async () => {
-  const expectedOutput =
-    "Congrats, you're up and running! Welcome to the first moshtix code challenge! Please replace this message with your solution and feel free to add or modify other files as necessary. Good luck : )";
+test("number writer working", async () => {
 
   logger.log = jest.fn();
 
   await go();
 
-  expect(logger.log).toBeCalledWith(expectedOutput);
+  expect(logger.log).toHaveBeenCalledWith(1);
```

Comment:
Tests shouldn't have any knowledge of the internal workings of the function being tested and should test isolated logic. So you need to break you solution down into functions so that you can test the function by itself. These tests don't really prove that the program is working correctly, just that at some point each word was logged, not necessarily in the right cases. So think about simple inputs and outputs that can be tested and consider writing your tests first to ensure your code is testable and the 'api' of your function/module is consumer friendly i.e. `when I call getBossHogResult(2) then should return 2` and `when I call getBossHogResult(3) then should return 'Boss'` etc.